action.yml

```diff
@@ -71,6 +71,8 @@ runs:
       working-directory: ${{ inputs.package-folder }}
       run: |
         dotnet pack \
+          --include-symbols \
+          --include-source \
           --configuration Release \
```

Comment:
maybe we will need also upload additional source file - Dušan Baník will test it

Reply:
maybe we can do it optionally so users can decide whethere they want inlude symbols and/or sources